.gitignore

```diff
@@ -8,3 +8,4 @@ Makefile.in
 autom4te.cache
 .DS_Store
 *~
+/doc
```

Comment:
Missing newline at end of file. Also, this change appears to be unrelated to the purpose of this PR. Could you give some context as to what tool is producing this `/doc` directory and open a separate PR to change it if needed?